modules/configuring-vsphere-connection-details.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-connection-details_{context}"]
+= Configuring vSphere connection details
+
+[role="_abstract"]
+Update the following vSphere configuration settings:
+
+* vCenter network address
+* vCenter username
+* vCenter password
+* vCenter address
+* vSphere data center
+* vSphere datastore
+* Virtual machine of the cluster
+
+.Prerequisites
+* The {ai-full} has finished installing the cluster successfully.
+* The cluster is connected to link:https://console.redhat.com/.
+
+.Procedure
+. In the Administrator perspective, navigate to *Home > Overview*.
+. Under *Status*, click *vSphere connection*. This opens the *vSphere connection configuration* wizard.
+. In the *vCenter* field, enter the network address of the vSphere vCenter server. This can be either a domain name or an IP address. It appears in the vSphere web client URL; for example `https://[your_vCenter_address]/ui`. 
+. In the *Username* field, enter the base64-encoded version of your vSphere vCenter username.
```

Comment:
Both the `username` or `password` should be provided encoded, the real human-readable values. The base64 envoding is up to the application to do (plugin)

Reply:
Do you know where the username is stored? Also in the kube-system namespace?

Reply:
The plugin stores them into secret name: `vsphere-creds`, namespace: `kube-system`.

Reply:
I am assuming the password is also stored here, correct?